src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result = this[0];
+    let i = 1;
```

Comment:
Use more descriptive names 

```suggestion
    let accumulator = this[0];
    let startIndex = 1;
```